pages/common/diskonaut.md

```diff
@@ -0,0 +1,20 @@
+# diskonaut
+
+> TUI disk space navigator written in rust.
+> More information: <https://github.com/imsnif/diskonaut>.
+
+- Start diskonaut in currenty directory:
+
+`diskonaut`
+
+- Start diskonaut for given directory:
```

Comment:
Usually we use 'specific' rather than 'given'.
```suggestion
- Start diskonaut in a specific directory:
```

Reply:
Good to know ^^